From 03aa92749d96dd03b833eb1ccedcabe763a8322b Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Mon, 21 Jan 2013 13:30:25 +0800
Subject: [PATCH 086/231] ENGR00240972-1 IPU: Add deinterlace frame rate
 double flags

Add deinterlace frame rate double flags for ipu header file

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 include/linux/ipu.h |   10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/include/linux/ipu.h b/include/linux/ipu.h
index f8178f1..ef55d80 100644
--- a/include/linux/ipu.h
+++ b/include/linux/ipu.h
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2012 Freescale Semiconductor, Inc.
+ * Copyright 2005-2013 Freescale Semiconductor, Inc.
  */
 
 /*
@@ -168,6 +168,14 @@ struct ipu_deinterlace {
 	u8	motion; /*see ipu_motion_sel*/
 #define IPU_DEINTERLACE_FIELD_TOP	0
 #define IPU_DEINTERLACE_FIELD_BOTTOM	1
+#define IPU_DEINTERLACE_FIELD_MASK	\
+		(IPU_DEINTERLACE_FIELD_TOP | IPU_DEINTERLACE_FIELD_BOTTOM)
+	/* deinterlace frame rate double flags */
+#define IPU_DEINTERLACE_RATE_EN		0x80
+#define IPU_DEINTERLACE_RATE_FRAME1	0x40
+#define IPU_DEINTERLACE_RATE_MASK	\
+		(IPU_DEINTERLACE_RATE_EN | IPU_DEINTERLACE_RATE_FRAME1)
+#define IPU_DEINTERLACE_MAX_FRAME	2
 	u8	field_fmt;
 };
 
-- 
1.7.10.4

